test-suite/TestSpec.hs

```diff
@@ -171,6 +173,30 @@ spec = do
         describe "concatPadLine" $ do
             it "even" $ concatPadLine 9 (Line 9 3 ["It", "is", "on"]) `shouldBe` "It is on"
             it "odd" $ concatPadLine 13 (Line 11 4 ["It", "is", "on", "us"]) `shouldBe` "It  is on  us"
+
+    describe "formatted text" $ do
+        let multiformat = multiFormatted "XXX" [plain "Hello", multiFormatted "Z" [plain "there"] "W"] "YYY"
```

Comment:
I'm not a fan of the name.  Would `example`, `exampleMF`, or something like that be ok with you?